mlir/lib/Conversion/TosaToMIOpen/TosaToMIOpen.cpp

```diff
@@ -63,20 +81,30 @@ class ConvConverter final : public OpConversionPattern<tosa::Conv2DOp> {
       return Value();
     }
     auto shape = oprType.getShape();
-    SmallVector<int64_t, 5> expShape(shape.begin(), shape.end());
-    expShape.push_back(1);
-    auto newType = MemRefType::get(expShape, oprType.getElementType());
+    SmallVector<int64_t, 5> expShape;
```

Comment:
I made some similar updates to this routine in PR https://github.com/ROCmSoftwarePlatform/llvm-project-mlir/pull/527 but I will wait for your PR and merge then.